src/components/videoPage/VideoAIMetadataContent.tsx

```diff
@@ -0,0 +1,50 @@
+import { LoadingOutlined } from '@ant-design/icons';
+import s from 'src/components/videoPage/style.module.less';
+import Badge from '@boclips-ui/badge';
+import { Typography } from '@boclips-ui/typography';
+import React from 'react';
+
+interface Props {
+  isLoading: boolean;
+  metadata?: string[];
+  type: string;
```

Comment:
What do you think about using an enum for the type? Enums make code more readable because you can readily see the values that are available.

Reply:
Good catch  ! I actually changed it to enum but didn't push it :')